addons/website_slides/tests/test_ui_wslides.py

```diff
@@ -148,6 +148,25 @@ def test_full_screen_edition_website_publisher(self):
             'odoo.__DEBUG__.services["web_tour.tour"].tours.full_screen_web_editor.ready',
             login=user_demo.login)
 
+    def test_course_reviews_elearning_officer(self):
+        user_demo = self.user_demo
+        user_demo.flush()
```

Comment:
why do you need to flush ?

Reply:
Actually I copy/pasted the code above and didn't think about it ^^
I removed it